docs/man_pages/project/testing/appstore.md

```diff
@@ -0,0 +1,48 @@
+appstore
+==========
+
+Usage | Synopsis
+---|---
+General | `$ tns appstore [<Apple ID> [<Password>]]`
+<% if((isConsole && isMacOS) || isHtml) { %>General | `$ tns appstore <Command>`<% } %>
```

Comment:
Since we have only one command here, you can probably forgo the Command part and replace it with the command itself. In this case, change General to Publish or Upload